src/sdlsound.cpp

```diff
@@ -101,7 +101,8 @@ bool init_sound()
     // We should only need to init once
     if( !sound_init_success ) {
         // Mix_OpenAudio returns non-zero if something went wrong trying to open the device
-        if( !Mix_OpenAudio( audio_rate, audio_format, audio_channels, audio_buffers ) ) {
+        if( !Mix_OpenAudioDevice( audio_rate, audio_format, audio_channels, audio_buffers, nullptr,
+            SDL_AUDIO_ALLOW_FREQUENCY_CHANGE ) ) {
```

Comment:
```suggestion
        if( !Mix_OpenAudioDevice( audio_rate, audio_format, audio_channels, audio_buffers, nullptr,
                                  SDL_AUDIO_ALLOW_FREQUENCY_CHANGE ) ) {
```

Reply:
Thanks for help, I'm still getting lost on github.